src/popup/components/Modals/TransferSend.vue

```diff
@@ -34,8 +34,12 @@
       <BtnMain
         class="button-action-primary"
         :disabled="error || !isConnected || !transferData.address || !transferData.amount"
-        :icon="showSendButton ? ArrowSendIcon : null"
-        :text="showSendButton ? $t('pages.send.send') : $t('modals.send.next')"
+        :icon="(showSendButton && !isMultisig) ? ArrowSendIcon : null"
+        :text="showSendButton
+          ? isMultisig
+            ? $t('modals.multisigTxProposal.proposeAndApprove')
+            : $t('pages.send.send')
+          : $t('modals.send.next')"
         data-cy="next-step-button"
         @click="proceedToNextStep"
       />
```

Comment:
If I'm not mistaken we do have two "Send Buttons" - I'd suggest wrapping the whole logic in a component and use it in both places (Account Details & Dashboard)